### DISCUSSION TITLE
Storage Information Message

### DISCUSSION THREAD

**Discussion by Oisin-JK** [opengisch/QField#3098] [Q&A]
It is possible to turn off the Storage Information message that appears when opeing a local project in QFIeld?

I've read it already.

**Comment by lucienicolier** [comment]
Hi,

do you have a screenshot of the message? 
Thanks

**Reply by Oisin-JK** [reply]
Hi
Here is a screenshot of the message. It appears whenever I press the "open a local file" button. I have followed the link, but it doesn't stop it appearing. Just a bit annoying really.

![Screenshot_20220726-213446](https://user-images.githubusercontent.com/57039094/181108462-ce9a81bf-6521-4448-b679-a4891311b665.jpg)

**Comment by suricactus** [comment]
@Oisin-JK Please update to QField 2.1.1 or newer.
